### PR TITLE
Add helpers to easily read and write a serializable type to JSON

### PR DESCRIPTION
**Pull request by DanilaFe**
Resolves https://github.com/chapel-lang/chapel/issues/8278.

This PR adds methods to convert types to and from JSON strings. These serve as convenience wrappers for the existing JSON (de)serialization functionality implemented by @benharsh in https://github.com/chapel-lang/chapel/pull/22437.

```Chapel
use IO, JSON;

record myPair : serializable {
  var x: int;
  var y: string;
}

var myValue = new myPair(1, "hello");
var myValueJson = toJson(myValue);

writeln(myValueJson);
writeln(fromJson(myValueJson, myPair));
```

Prints:

```
{"x":1, "y":"hello"}
(x = 1, y = hello)
```

Reviewed by @jeremiah-corrado -- thanks!

## Testing
- [x] paratest

### CODE REVIEW

File: test/io/serializers/jsonHelpers/basicread.chpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+use IO, JSON;
+
+record myPair : serializable {
+  var x: int;
+  var y: string;
+}
+
+var myValue = new myPair(1, "hello");
+var myValueJson = toJson(myValue);
```
Comment (jeremiah-corrado) [inline test/io/serializers/jsonHelpers/basicread.chpl:9]: these values are unused, should they be removed from this test?

**Review by jeremiah-corrado — APPROVED** [review]
looks good!

File: test/io/serializers/jsonHelpers/basicread.chpl (hunk 1 above)
Comment (DanilaFe) [reply test/io/serializers/jsonHelpers/basicread.chpl]: Good catch, they should be.

**Comment by bradcray** [comment]
I don't know much about JSON, but this seems very cool and productive from a distance, thanks!

**Pull request merged**